apps/backend/api/models/Post.js

```diff
@@ -1211,6 +1256,76 @@ module.exports = bookshelf.Model.extend(Object.assign({
 
     // Use Queue system for non-blocking post subscription publishing
     Queue.classMethod('Post', 'publishPostUpdates', { postId, options }, 0)
-  }
+  },
 
+  sendEventUpdateRsvps: async function ({ postId, eventChanges }) {
+    const post = await Post.find(postId)
+    const eventInvitations = await post.eventInvitations().fetch()
+
+    eventInvitations.forEach(eventInvitation => {
+      if (!eventInvitation.notGoing()) {
+        Post.sendEventRsvp({eventId: postId, eventInvitationId: eventInvitation.id, eventChanges})
+      }
+    })
+  },
+
+  sendEventCancelRsvps: async function ({ postId, eventInvitationIds }) {
+    const post = await Post.where({ id: postId }).fetch() // post is likely deactive, so fetch manuely
+    for (const eventInvitationId of eventInvitationIds) {
```

Comment:
I think we should load these from the post here, not pass them in. what if they changed in the time between when the job was queued and called?